src/main/java/org/maxgamer/quickshop/util/envcheck/EnvironmentChecker.java

```diff
@@ -48,10 +49,12 @@
 public class EnvironmentChecker {
     private final QuickShop plugin;
     private final List<Method> tests = new ArrayList<>();
+    private final List<String> startupArgs;
 
     public EnvironmentChecker(QuickShop plugin) {
         this.plugin = plugin;
         this.registerTests(this.getClass()); //register self
+        this.startupArgs = ManagementFactory.getRuntimeMXBean().getInputArguments();
```

Comment:
We should check the system properties, not the args.

Reply:
No I just want some args not all, it is useless so only check input arguments